src/Settings/InformationArchitecture.php

```diff
@@ -76,20 +58,8 @@ public static function get_fields(): array {
 	 * @return bool Whether the option is active.
 	 */
 	public static function is_active( string $name ): bool {
-		return ! empty( self::get( $name ) );
-	}
-
-	/**
-	 * Return option value.
-	 *
-	 * @param string $name    Name of the option.
-	 * @param mixed  $default Default value.
-	 *
-	 * @return mixed
-	 */
-	public static function get( string $name, $default = null ) {
 		$options = get_option( self::OPTIONS_KEY );
 
-		return isset( $options[ $name ] ) ? $options[ $name ] : $default;
+		return 'on' === $options[ $name ];
```

Comment:
I had to preserve this function because the plugin calls it. _Whispers monorepo_